parts/linux/cloud-init/artifacts/containerd-monitor.timer

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=a timer that delays containerd-monitor from starting too soon after boot
+[Timer]
+OnBootSec=30min
```

Comment:
would this be too long?

Reply:
this is what we have set for docker monitor. we could consider trimming both down a bit.

Reply:
shortened to 10mins.